suite-common/wallet-utils/src/localizeNumber.ts

```diff
@@ -21,7 +21,7 @@ export const localizeNumber = (
     const amountRoundedDown = amount.toFixed(0, BigNumber.ROUND_DOWN);
 
     const wholeNumber = BigInt(amountRoundedDown);
-    const formattedWholeNumber = Intl.NumberFormat(locale).format(wholeNumber);
+    const formattedWholeNumber = Intl.NumberFormat(locale).format(Number(wholeNumber));
```

Comment:
This change causes big integers to display incorrectly and breaks some tests, why is it needed @bouchja1 ?

Reply:
The change is included in cca8504a1c8247c6d0dd1612e1a158954cf04f78, seems to be working. @tomasklim FYI